citadel/install_ubuntu_src.md

```diff
@@ -28,6 +28,7 @@ pip install vcstool
 ```bash
 pip install -U colcon-common-extensions
 ```
+Test the programs installed using PIP, if the system is not recognising the commans, and you're using a sistem that is compatible with Debian or Ubuntu packages, **install using apt is a good idea because the system is automatically configured**.  
```

Comment:
> Test the programs installed using PIP

A user who doesn't know how these programs work wouldn't know how to test them. How about saying something like:

```suggestion
Check that no errors were printed while installing with PIP. If your system is not recognising the commands, and you're using a system that is compatible with Debian or Ubuntu packages, see the instructions below to install using `apt`.  
```

Reply:
I liked it, better, thanks!